website/documentation/getting-started/shoots/_index.md

```diff
@@ -0,0 +1,126 @@
+
+![](./images/basic_configurations_4.png)
+
+Once per day, all clusters reconcile. This means all controllers will check if there are any updates they have to apply (i.e. new image version for ETCD). The maintenance window defines when this daily operation will be triggered. It is important to understand that there is no opt-out for reconciliation.
+
+It is also possible to confine updates to the shoot spec to be applied only during this time. This can come in handy when you want to bundle changes or prevent changes to be applied outside a well-known time window.
+
+You can allow Gardener to automatically update your cluster's Kubernetes patch version and/or OS version (of the nodes). Take this decision consciously! Whenever a new Kubernetes patch version or OS version is set to `supported` in the respective cloud profile, auto update will upgrade your cluster during the next maintenance window. If you fail to (manually) upgrade the Kubernetes or OS version before they expire, force-upgrades will take place during the maintenance window.
+
+### Result
+
+The result of user configuration and defaulting is a shoot resource that, once applied, will be acted upon by various Gardener components. The status section represents the intermediate steps / results of these operations. A typical shoot creation flow would look like this:
+
+1. Assign control plane to a seed.
+1. Create infrastructure resources in the data plane account (e.g., VPC, gateways, ...)
+1. Deploy control plane incl. DNS records.
+1. Create nodes (VMs) and bootstrap kubelets.
+1. Deploy kube-system components to nodes.
+
+## How to Access a Shoot
+
+![](./images/access-shoot.png)
+
+Static credentials for shoots are discontinued with Kubernetes v1.27 in Gardener. Use short lived credentials instead. You can create/request tokens directly via Gardener or delegate authentication to an identity provider.
+
+A short-lived admin kubeconfig can be requested by using kubectl. If this is something you do frequently, consider switching to gardentctl (v2), which helps with it.
+
+![](./images/access-shoot-2.png)
+
+An alternative is to use an identity provider and issue oidc tokens.
+
+## What can you configure?
+
+With the basic configuration options known, it is time to discuss more possibilities. Gardener offers a variety of options to tweak the control plane's behavior - like defining an event TTL (default 1h), adding an OIDC configuration or activating some feature gate. You can alter the scheduling profile and define an audit logging policy. In addition, the control plane can be configured to run in HA mode (applied on a node or zone level), but note that it cannot be changed.
```

Comment:
Could the final sentence be changed from `keep in mind that once you **went** HA, you cannot go back.` to `keep in mind that once you **enable** HA, you cannot go back.`? If it isn't factually incorrect, think it would sound better.